components/k8s-model-server/http-proxy/README.md

```diff
@@ -43,5 +43,47 @@ or
 
 - **Input and Output Schema**:
 
-In the example above, that correspones to a [tensorflow savedModel](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/saved_model/README.md) with input signature `"tf_model_input"` and `"image"` and output signature `"tf_model_output"`.
+In the example above, that correspones to a [tensorflow savedModel](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/saved_model/README.md) with method signature `tf.saved_model.signature_constants.PREDICT_METHOD_NAME`, input signature `"tf_model_input"` and `"image"` and output signature `"tf_model_output"`.
```

Comment:
Which example are you referring to?

Reply:
the rest api example. i'll add a link to that